components/visual-crypto/VisualSecretSharing.tsx

```
@@ -0,0 +1,336 @@
+            )}
+          </div>
+
+          <div className="mt-4 flex flex-wrap items-center justify-between gap-2">
+            <div className="flex items-center gap-2">
+              <span className="text-xs font-semibold text-zinc-500">Presets:</span>
+              <button
+                type="button"
+                onClick={() => loadPreset('C')}
+                className="rounded bg-zinc-100 px-2 py-1 text-xs font-medium text-zinc-700 hover:bg-zinc-200 dark:bg-zinc-800 dark:text-zinc-300"
+              >
+                'C'
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)